app/views/provider_management/external_users/_form.html.haml

```diff
@@ -1,60 +1,34 @@
 
   = f.fields_for :user do |user_fields|
-    .form-fields
-      %fieldset
-        %legend.visually-hidden New user
-
-        .form-group
-          = user_fields.label :first_name, t('.first_name'), class: 'form-label'
-          = user_fields.text_field :first_name, class: 'form-control'
-
-        .form-group
-          = user_fields.label :last_name, t('.last_name'), class: 'form-label'
-          = user_fields.text_field :last_name, class: 'form-control'
-
-        .form-group
-          = user_fields.label :email, t('.email'), class: 'form-label'
-          = user_fields.email_field :email, class: 'form-control'
-
-        .form-group
-          = user_fields.label :email_confirmation, t('.email_confirmation'), class: 'form-label'
-          = user_fields.email_field :email_confirmation, class: 'form-control'
-
-      %fieldset.inline{ 'aria-describedby': 'checkbox-control-legend-roles' }
-        %legend#checkbox-control-legend-roles
-          %span.form-label-bold
-            = t('.roles')
-
-        .form-group.inline.js-advocate-roles
-          = f.collection_check_boxes(:roles, @external_user.available_roles, :to_s, :to_s) do |b|
-            .multiple-choice
-              = b.check_box
-              = b.label(id: b.value.to_css_class) { b.value.humanize }
-
-
-      - if @provider.chamber?
-        .form-group
-          %fieldset.inline
-            %legend
-              %span.form-label-bold
-                = t('.apply_vat')
-              = f.collection_radio_buttons(:vat_registered, [true, false], :to_s, :to_s) do |b|
-                .multiple-choice
-                  = b.radio_button
-                  = b.label(id: b.value.to_css_class) { (b.value.to_s == 'true' ? 'Yes' : 'No') }
-
-        .form-group
-          = f.label :supplier_number, t('.supplier_number'), class: 'form-label'
-          = f.text_field :supplier_number, class: 'form-control'
-
-    .form-group
-      = govuk_button(t(".submit.#{action_name}"))
+    = user_fields.govuk_error_summary
+
+    = user_fields.govuk_text_field :first_name,
+      label: { text: t('.first_name') }
+
+    = user_fields.govuk_text_field :last_name,
+      label: { text: t('.last_name') }
+
+    = user_fields.govuk_email_field :email,
+      label: { text: t('.email') }
+
+    = user_fields.govuk_email_field :email_confirmation,
+      label: { text: t('.email_confirmation') }
+
+    = f.govuk_collection_check_boxes :roles,
+      @external_user.available_roles,
+      :to_s,
+      :humanize,
+      legend: { text: t('.roles'), size: 's' }
+
+    - if @provider.chamber?
+      = f.govuk_collection_radio_buttons :vat_registered,
+        [['Yes','true'],['No','false']],
+        :last,
+        :first,
+        legend: { text: t('.apply_vat'), size: 's' }
+
+      = f.govuk_text_field :supplier_number,
+        label: { text: t('.supplier_number') }
```

Comment:
For discussion only: looks like this is like-for-like with current functionality so good in that respect, but are we doing the same thing differently on the provider side of things  - i.e. nesting these fields under the advocates checkbox.